### PR TITLE
Delete old NL Interface - Finale

### PR DESCRIPTION
**Pull request by pradh**
After this change `/api/nl` is only used by Eval tool.

Switched the `nl_test.py` to use explore endpoint.

NOTE:  This changes the feedback endpoint, used only be JS, from `/api/nl/` to `/api/explore/`.

### CODE REVIEW

File: server/lib/nl/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+All NL related backend functionality is in this directory, organized as follows:
+
+* `detection/`: detects DCIDs and heuristics from NL query
+
+* `fulfillment/`: determines what content can be show
```
Comment (chejennifer) [inline server/lib/nl/README.md:5]: nit: "... can be shown"

File: server/lib/nl/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+* `config_builder/`: constructs charts with appropriate configs and titles 
+
+* `explore/`: includes additional Explore specific functionality (topic expansion, related things, etc.)
+
+* `common/`: miscellaneous utilities used by other modles
```
Comment (chejennifer) [inline server/lib/nl/README.md:11]: nit: modules

**Review by chejennifer — APPROVED** [review]
nice!

File: server/lib/nl/README.md (hunk 1 above)
Comment (pradh) [reply server/lib/nl/README.md]: Done

File: server/lib/nl/README.md (hunk 2 above)
Comment (pradh) [reply server/lib/nl/README.md]: Done

**Comment by pradh** [comment]
Thanks for the review!

**Pull request merged**